### DISCUSSION TITLE
Particular audio file fails with API transcription but works with web app

### DISCUSSION THREAD

**Discussion by mcg-wellsaid** [deepgram/community#483] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

I have a particular audio file (I can potentially share it if desired, but need to get internal clearance first) that I am able to transcribe using the web app. But when I try to use the Python SDK method `deepgram.transcription.sync_prerecorded` to hit the API, it fails. I get an error like this:

```
DeepgramApiError: Request `5e52dc63-4912-40ea-a4ec-9937e8961c22` returned 400: Bad Request: Invalid data received.
```

It doesn't matter if the request is using the `buffer` or `url` modality, same result. I am able to make a successful request with a different audio file (`interview_speech-analytics.wav` that is linked in the docs).

I am wondering what is up with this particular file I am trying that is causing the error? How can I debug this?

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

5e52dc63-4912-40ea-a4ec-9937e8961c22

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by mcg-wellsaid** [comment]
Here's a snippet showing how I formulate the request:

```
deepgram = Deepgram(DEEPGRAM_API_KEY)

with open('abbi_d__narration_1.wav', 'rb') as audio:
    source = {
        'buffer': audio,
        'mimetype': 'audio/wav',
    }
    
    response = deepgram.transcription.sync_prerecorded(
      source,
      {
        'smart_format': True,
        'model': 'nova-2',
      }
    )
    
    print(response)
```

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @mcg-wellsaid, this may be a case of Deepgram not handling your 24-bit audio (`pcm_s24le`) well. Can you please use 16-bit encoding (`pcm_s16le`) and retry your request with that new audio file? Your code looks fine.

You can re-encode your audio file using the command: `ffmpeg -i <in>.wav -acodec pcm_s16le <out>.wav`

**Comment by mcg-wellsaid** [comment]
Thanks. It works with 16-bit audio. Would be great if requirements like that were listed in the docs.

**Reply by jkroll-deepgram** [reply]
Glad to hear that works for you. Apologies for the poor UX; this is a bug on our side. We support a [wide variety of audio formats](https://developers.deepgram.com/docs/supported-audio-formats), and intend to support the `pcm_s24le`, but in particular that format can cause trouble. 16 or 32 bit audio is more typical than 24 bit.
